Classes/KIFTestActor.m

```diff
@@ -179,7 +180,13 @@ - (void)waitForTimeInterval:(NSTimeInterval)timeInterval
 }
 
 -(void)waitForAnimationsToFinish {
```

Comment:
Animations are a UI concept only, so I'd expect this would make more sense living in KIFUITestActor than KIFTestActor.